exe/kubernetes-deploy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require 'kubernetes-deploy'
+
+options = {
+  namespace: ARGV[0],
+  context: ARGV[1],
+  environment: ENV['ENVIRONMENT'],
+  current_sha: ENV['REVISION'],
+  kubeconfig_path: ENV["KUBECONFIG"],
+}
+
+options[:template_dir] = ENV['K8S_TEMPLATE_FOLDER'] || File.expand_path("./config/deploy/#{options.fetch(:environment)}")
```

Comment:
`$KUBECONFIG` is already handled by k8s as-is: https://kubernetes.io/docs/user-guide/kubeconfig-file/#loading-and-merging-rules.

`config/deploy/{env}` is where we drop the files for non-core apps on cloud init.